src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -16,4 +19,19 @@ describe('elements/content-sidebar/BoxAISidebar', () => {
 
         expect(screen.getByRole('heading', { level: 3, name: 'Box AI' })).toBeInTheDocument();
     });
+
+    test('should have accessible "Expand" button', () => {
+        renderComponent();
+
+        expect(screen.getByRole('button', { name: 'Expand' })).toBeInTheDocument();
+    });
+
+    test('should call onExpandClick when click "Expand" button', async () => {
+        renderComponent();
+
+        const expandButton = screen.getByRole('button', { name: 'Expand' });
+        await userEvent.click(expandButton);
+
+        expect(mockOnExpandClick).toHaveBeenCalled();
```

Comment:
there can be issues and unexpected behavior if we forget to clear/reset the mock calls between tests. i think we handle this in a global configuration but just in case, it's better to not use reuse the fn mock for the `expect` so it's more intentional about what should happen

e.g. check for a separate prop (defined in the local scope) and/or explicitly add an `afterEach`
```
afterEach(() => {
    jest.clearAllMocks();
});

// or

const onExpandClick = jest.fn();
renderComponent({ onExpandClick });

...

expect(onExpandClick).toHaveBeenCalled();
```